server/zanata-war/src/test/java/org/zanata/adapter/PropertiesAbstractTest.java

```diff
@@ -45,16 +45,11 @@ TranslationsResource addTranslation(TranslationsResource resource,
     }
 
     File createTempFile(Charset charset) throws Exception {
-        File testFile;
-        try {
-            testFile = File.createTempFile("test-properties-latin1", "properties");
-        } catch (IOException ioException) {
-            throw new RuntimeException("Unable to create temporary file ");
-        }
+        File testFile = File.createTempFile("test-properties-temp", "properties");
         assertThat(testFile.exists());
         Map<String, String> entries = new HashMap<>();
         if (charset == StandardCharsets.ISO_8859_1) {
-            entries.put("line1", "Line One");
+            entries.put("line1", "^Line One");
```

Comment:
I don't think `circumflex accent` is the ideal choice, because it seems to have the same representation in UTF-8. Try `À` which is `0xC0` in Latin-1. `0xC0` is illegal in UTF-8.